efficientdet/keras/train.py

```diff
@@ -62,6 +62,9 @@
     'and this flag has no effect.')
 flags.DEFINE_string('model_dir', None, 'Location of model_dir')
 
+flags.DEFINE_string('ckpt', None,
```

Comment:
pretrained_ckpt

Reply:
Did you mean changing `ckpt` to `pretrained_ckpt`?

Reply:
yes

Reply:
BTW, the reason I used `ckpt` is because the term was used in tf 1.x training.
See:
https://github.com/google/automl/blob/3958f1b68a02b5e67075e6c45ecc2376131f4165/efficientdet/main.py#L53-L54

Reply:
Modified as suggested.